UnitTests/Tests/WeatherUpdateCacheSpec.swift

```diff
@@ -0,0 +1,68 @@
+@testable import Tropos
+import Quick
+import Nimble
+
+private let testCacheFileName = "TestWeatherUpdate"
+
+private var testCachesDirectory: NSURL {
+    return NSFileManager.defaultManager()
+        .URLsForDirectory(.DocumentDirectory, inDomains: .UserDomainMask)
+        .first!
+}
+
+private var testWeatherUpdateURL: NSURL {
+    return testCachesDirectory.URLByAppendingPathComponent(testCacheFileName)
+}
+
+private func resetFilesystem() {
+    do {
```

Comment:
When I was testing this, `removeItemAtURL(_:)` would throw if you tried to remove a path that doesn't exist. So this is essentially `rm -f`, where if the file already doesn't exist it's no problem.